Docs/Configuration.tex

```diff
@@ -3653,16 +3651,16 @@ \subsection{Security Properties}\label{miscsecurityprops}
   allowing to enabling Apple Secure Boot with any SMBIOS. Setting
   \texttt{SecureBootModel} to any valid value but \texttt{Disabled} is equivalent to
   \href{https://support.apple.com/en-us/HT208330}{\texttt{Medium Security}}
-  of Apple Secure Boot. To achieve \texttt{Full Security} one will need to
-  also specify \texttt{ApECID} value.
+  of Apple Secure Boot. The \texttt{ApECID} value must also be specified to
+  achieve \texttt{Full Security}.
 
   Enabling Apple Secure Boot is more demanding to incorrect configurations,
-  buggy macOS installations, and unsupported setups. Things to keep in mind:
+  buggy macOS installations, and unsupported setups. Things to consider:
```

Comment:
*keep in mind* should be fine here I guess? (i.e. to put emphasis)

Reply:
Don't think so.

Reply:
Yeah, I've sort of overdone with the informal things. Not to be blamed for too much for writing the whole thing though.